lib/src/mock_navigator.dart

```diff
@@ -1,7 +1,3 @@
-/// An experimental package that attempts to make it easy to mock Flutter's
```

Comment:
Should we rename this file to `mockingjay.dart`?

Reply:
It has, but GitHub is showing it as a moved file with some diffs since now we have a `lib/src/mock_navigator.dart`. If we rename that file to `mockingjay.dart` as you mentioned, then the diff will be less ugly :)